server/init.go

```diff
@@ -118,6 +80,54 @@ func GenTxCmd(ctx *Context, cdc *wire.Codec, appInit AppInit) *cobra.Command {
 	return cmd
 }
 
+func gentxWithConfig(config *cfg.Config, gaiaConfig *gaiacfg.Config, ctx *Context, cdc *wire.Codec, appInit AppInit) (
```

Comment:
convention, `ctx` should always be the first input to a function if it exists

Reply:
also let's not use pointers for the config's if possible

Reply:
I fixed the `ctx` convention, but in the Tendermint code, every function that handles the config is expecting a pointer. If I modify it here, my code will be full with &config horribleness.

For example, a few lines below, I'll have to change this:
```
	pubKey := readOrCreatePrivValidator(config)
```
to this:
```
	pubKey := readOrCreatePrivValidator(&config)
```